scripts/google_compliance.py

```diff
@@ -0,0 +1,68 @@
+"""
+End-to-end tests that run against the Google server
+"""
+
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import subprocess
+
+
+def runTests():
+    separator = "----------------------"
+    path = os.path.join(
+        os.path.abspath(os.path.dirname(__file__)),
+        '..', 'client_dev.py')
+    baseUrl = "https://www.googleapis.com/genomics/v1beta2"
+    keyStr = "--key AIzaSyBcdyeUZSzUVE0e_HPLyCJd7bOZ6g3lr3I"
+    workarounds = "--workarounds=google"
+    commands = [
+        # TODO get example that works
+        # ("variants-search", "1000g_2013", "-r1 -e 500 -c HG03279"),
+        "variantsets-search --datasetIds 10473108253681171589",
+        "referencesets-search --accessions GCA_000001405.15",
+        "references-search --md5checksums 1b22b98cdeb4a9304cb5d48026a85128",
+        "readgroupsets-search --datasetIds 10473108253681171589 "
+        "--name NA12878 --pageSize 1",
+        "callsets-search --variantSetIds 10473108253681171589 "
+        "--name HG00261 --pageSize 1",
+        "reads-search --start 51005353 --end 51005354 --readGroupIds "
+        "ChhDTXZuaHBLVEZoQ2JpT2J1enZtN25Pb0IQAA --referenceName "
+        "22 --pageSize 1",
+    ]
+    for command in commands:
+        if isinstance(command, tuple):
```

Comment:
I think this only exists for the "variants-search" which is commented out above? It would be nice to clean this up and keep a consistent interface for all the commands before we merge. I appreciate this is stuff you're trying to get working though.